packages/core/src/components/QueryBuilder/RuleGroup/RuleGroup.tsx

```diff
@@ -0,0 +1,337 @@
+                ? labels.query?.addRule?.label
+                : labels.group.addRule.label}
+            </HvButton>
+          </div>
+          {level <= normalizedMaxDepth && (
+            <div
+              className={cx(
+                queryBuilderClasses.buttonBackground,
+                css(styles.buttonBackground, classes?.buttonBackground)
+              )}
+            >
+              <HvButton
+                variant="secondarySubtle"
+                onClick={() => {
+                  dispatchAction({ type: "add-group", id });
+                }}
+                disabled={readOnly}
+                startIcon={<Add />}
+              >
+                {level === 0 && labels.query?.addGroup?.label != null
+                  ? labels.query?.addGroup?.label
+                  : labels.group.addGroup.label}
+              </HvButton>
+            </div>
+          )}
+        </>
+      )}
+    </ClassNames>
+  );
+
+  const DeleteIcon = withTooltip(
+    () => (
+      <ClassNames>
+        {({ css, cx }) => (
+          <Delete
+            className={
+              readOnly
+                ? cx(
+                    queryBuilderClasses.topRemoveButtonDisabled,
+                    css(
+                      styles.topRemoveButtonDisabled,
+                      classes?.topRemoveButtonDisabled
+                    )
+                  )
+                : ""
+            }
+          />
+        )}
+      </ClassNames>
+    ),
+    level === 0 && labels.query?.delete?.tooltip
+      ? labels.query?.delete?.tooltip
+      : labels.group.delete.tooltip,
+    "top"
+  );
+
+  const onClickCombinator = useCallback(
+    (item) => {
+      dispatchAction({
+        type: "set-combinator",
+        id,
+        combinator: item.operand,
+      });
+    },
+    [dispatchAction, id]
+  );
+
+  return (
+    <ClassNames>
+      {({ css, cx }) => (
+        <div
+          className={cx(
+            queryBuilderClasses.root,
+            css(styles.root),
+            classes?.root,
+            level === 0
+              ? cx(
+                  queryBuilderClasses.topGroup,
+                  css(styles.topGroup),
+                  classes?.topGroup
+                )
+              : cx(
+                  queryBuilderClasses.subGroup,
+                  css(styles.subGroup),
+                  classes?.subGroup
+                )
+          )}
+        >
+          <HvGrid container>
+            <HvGrid item>
+              <HvMultiButton
+                className={cx(
+                  queryBuilderClasses.combinator,
+                  css(styles.combinator, classes?.combinator),
+                  queryBuilderClasses.topCombinator,
+                  css(styles.topCombinator, classes?.topCombinator)
+                )}
+                disabled={readOnly}
+              >
+                {combinators.map((item) => (
+                  <HvButton
+                    key={item.operand}
+                    className={cx(
```

Comment:
Should be:
`cx(queryBuilderClasses.combinatorButton, css(styles.combinatorButton), classes?.combinatorButton)`